### DISCUSSION TITLE
Error when trying to resolve target in glsp-examples/project-templates/java-emf-eclipse

### DISCUSSION THREAD

**Discussion by madamedefarge** [eclipse-glsp/glsp#1316] [General]
I am following the README in the above project file to set the target platform. 
I am using the target definition at;-
org.eclipse.glsp.example.javaemf.parent/org.eclipse.glsp.example.javaemf.target/r2023-09.target
but get an error message;-
No repository found containing: osgi.bundle,org.eclipse.emf.emfstore.common,1.7.0.v20160209-0511

It seems to be the repository entry for;-
<repository location="https://download.eclipse.org/glsp/ide/p2/releases/2.0.0/"/>
that is causing the error as the error disappears if I delete that entry but then the plugin org.eclipse.glsp.ide.editor is not resolved.
Thank you in advance for help.
<img width="1036" alt="Screenshot 2024-04-16 at 08 38 35" src="https://github.com/eclipse-glsp/glsp/assets/1956357/b7e59b2d-9cb7-4ec7-99dd-21d5e05b2899">

**Comment by tortmayr** [comment]
Hi @madamedefarge,
Could you please provide some more details about your system so that we can try to reproduce the error?
i.e OS, Eclipse Version, Java Version

**Comment by madamedefarge** [comment]
macos Silicon but I am running Eclipse 24-03 x86_64 with Rosetta. 
Java 17.0.10 with justj openjdk

**Comment by madamedefarge** [comment]
Just retried using Eclipse 23-09 x86_64 with Rosetta. Also Java 17. Target reload works.

Eclipse 23-12 x86_64 fails with same error.

**Comment by tortmayr** [comment]
@madamedefarge  Thanks for further investigating this. We will look into this and I have created #1317 to track this.

**Reply by madamedefarge** [reply]
thank you!
